<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sniffer_ui</class>
 <widget class="QMainWindow" name="sniffer_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>811</width>
    <height>597</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>811</width>
    <height>597</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>sniffer_ui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>219</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Destination</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Protocol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Length</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Info</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>801</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
